test/build/config-format/typescript/typescript.test.js

```diff
@@ -11,4 +15,19 @@ describe("webpack cli", () => {
     expect(exitCode).toBe(0);
     expect(existsSync(resolve(__dirname, "dist/foo.bundle.js"))).toBeTruthy();
   });
+
+  it('should work with the "disable-interpret" option from flags', async () => {
```

Comment:
Let's add another testcase where we haven't handled transpilation of config and also disabled `interpret`.

Reply:
@rishabh3112 

Yes, I added the test case : https://github.com/webpack/webpack-cli/pull/3329/commits/771f7a3c8647cd9c75521b496944c68219696ac6

Reply:
This looks good now 😄